pkg/encoding/bigint/bigint.go

```diff
@@ -79,15 +79,17 @@ func getEffectiveSize(buf []byte, isNeg bool) int {
 	return size
 }
 
-// IntToBytes converts integer to a slice in little-endian format.
+// ToBytes converts integer to a slice in little-endian format.
 // Note: NEO3 serialization differs from default C# BigInteger.ToByteArray()
 //   when n == 0. For zero is equal to empty slice in NEO3.
 // https://github.com/neo-project/neo-vm/blob/master/src/neo-vm/Types/Integer.cs#L16
-func IntToBytes(n *big.Int) []byte {
-	return intToBytes(n, []byte{})
+func ToBytes(n *big.Int) []byte {
+	return ToBytesSpecified(n, []byte{})
 }
 
-func intToBytes(n *big.Int, data []byte) []byte {
+// ToBytesSpecified converts integer to a slice in little-endian format using given
+// byte array for conversion result.
+func ToBytesSpecified(n *big.Int, data []byte) []byte {
```

Comment:
`ToBytesSlice`, `ToPreallocatedBytes`, `ToBytesWithBuffer`? @fyrchik? For some reason `ToBytesSpecified` doesn't sound right to me. Maybe it's just me.

Reply:
Changed the name to `ToPreallocatedBytes`